install/kubernetes/BUILD

```diff
@@ -2,8 +2,10 @@ filegroup(
     name = "kubernetes",
     srcs = [
         "istio.yaml",
-        "istio-auth.yaml",
```

Comment:
As mentioned over email I think we should keep a single file installing everything in 1 step (with auth)